lib/staccato/adapter/net_http_via_proxy.rb

```diff
@@ -0,0 +1,19 @@
+require 'net/http'
+
+module Staccato
+  module Adapter
+    class NetHttpViaProxy
+
+      def initialize(uri, proxy_host, proxy_port)
+        @uri = uri
+        @proxy_host = proxy_host
+        @proxy_port = proxy_port
+      end
+
+      def post(params)
+        ::Net::HTTP.Proxy(@proxy_host, @proxy_port).post_form @uri, params
```

Comment:
well, it's `.Proxy`, see http://ruby-doc.org/stdlib-2.3.1/libdoc/net/http/rdoc/Net/HTTP.html#method-c-Proxy

I've use this class in my project for months. After you suggest just use a separate adaptor for proxy, I just copied it here and removed keyword arguments.